### PR TITLE
SERV-263: Import statics

### PR DESCRIPTION
**Pull request by tylerlevine**

### CODE REVIEW

File: modules/statics/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (tylerlevine) [inline modules/statics/tsconfig.json]: extend root tsconfig

File: modules/statics/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# @bito/statics code owners and first officers will automatically be requested for review whenever a pull request touches the files they own.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners and first officers will be the default owners for everything in the repo.
+*   tyler@bitgo.com elle@bitgo.com
```
Comment (tylerlevine) [inline modules/statics/CODEOWNERS]: Incorporate into root codeowners

File: modules/statics/package-lock.json [hunk 3]
```diff
@@ -0,0 +1,2411 @@
+{
```
Comment (tylerlevine) [inline modules/statics/package-lock.json]: Remove

File: modules/statics/package.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  "types": "./dist/src/index.d.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc",
+    "fmt": "prettier --write 'src/**/*.ts'",
+    "check-fmt": "prettier --check 'src/**/*.ts'",
+    "clean": "rm -r ./dist"
+  },
+  "author": "",
+  "license": "Apache-2.0",
```
Comment (tylerlevine) [inline modules/statics/package.json]: Check license

File: modules/statics/package.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BitGo/statics"
```
Comment (tylerlevine) [inline modules/statics/package.json]: Update

File: modules/statics/scripts/publish.sh [hunk 6]
```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
```
Comment (tylerlevine) [inline modules/statics/scripts/publish.sh]: Check publish flow

**Comment by dannybitgo** [comment]
Should we change the express dockerfile to use node 10 as well instead of lts?

**Comment by tylerlevine** [comment]
> Should we change the express dockerfile to use node 10 as well instead of lts?

Yep, we should do that, let me update.

**Comment by tylerlevine** [comment]
> > Should we change the express dockerfile to use node 10 as well instead of lts?
> 
> Yep, we should do that, let me update.

Actually, I'm going to break out all the LTS changes into a separate PR.

**Pull request merged**